variablegroups/AAT-AKS-Common.json

```diff
@@ -28,6 +28,12 @@
       "value": "1.14.8"
     },
     "Subscription.AKS.nodeCount": {
+      "value": "2"
+    },
+    "Subscription.AKS.nodeMinCount": {
```

Comment:
It should be easier to do this with the arm template vars files:
https://github.com/hmcts/cnp-aks-rbac-pipelines/tree/master/templates/vars

You won't have the problem where this needs running on master first

Reply:
Yes this will be set via vars file only.